lib/renaissance/auctions/auction.ex

```diff
@@ -26,24 +26,22 @@ defmodule Renaissance.Auction do
   end
 
   defp validate_price(changeset) do
-    item_price = get_change(changeset, :price)
-    zero_dollars = Money.new(000, :USD)
-    one_dollar = Money.new(100, :USD)
+    {_, amount} = Money.Ecto.Amount.Type.dump(get_change(changeset, :price, 0))
 
-    if Money.compare(item_price || one_dollar, zero_dollars) == 1 do
+    if 0 < amount do
```

Comment:
could you reverse this ? `0 < amount` is harder to understand than `amount > 0` IMO

Reply:
Sure, will do.
_(I find it more intuitive to have the smaller value on the left, but don't feel strongly.)_